### ISSUE TITLE
Being  able to select a xsl file by URI

### ISSUE THREAD

**Issue by cmarchand** [cmarchand/xslCompiler-maven-plugin#5]
I want to be able to specify a XSL by any URI. Up to maven to load the URI, then...
Something like :
```xml
<filesets>
  <fileset>
    <uri>file:/home/cmarchand/devel/projects/test/src/main/xsl/test.xsl</uri>
  </fileset>
</filesets>
```

**Comment by cmarchand** [comment]
This solve has a bug.
It generates the compiled file in .../test.xsl/test.sef, it should be in .../test.sef

**Comment by cmarchand** [comment]
Solved in 1.0.3.
http://repo1.maven.org/maven2/top/marchand/xml/maven/xslcompiler-maven-plugin/1.0.3/
